### DISCUSSION TITLE
Can this be used with local LLM's?

### DISCUSSION THREAD

**Discussion by jadedgnome** [spdustin/ChatGPT-AutoExpert#95] [Q&A]
I wanted to know If and how i could use this with LLAMA model.

**Comment by spdustin** [comment]
With most models, not as written. Each model has its own quirks with how well it can attend to complex instructions.

Many models will benefit from an additional instruction that tells the model to:

> Begin by thinking about how your reply will meet the requirements specified in the instructions above. Share your thoughts inside `<thinking></thinking>` XML tags, then generate your reply. This is a private space for your thoughts, and will not be shown to {{replace with dialogue tag for user}}.

If the model's prompt format uses "Human" as its message tag for user messages, replace the {{}} text with "Human". If it uses "User", replace with "User", etc.
